src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/list.tsx

```diff
@@ -1,57 +1,149 @@
 import React from 'react';
 import styled from '@emotion/styled';
+import {
+  List,
+  ListRowProps,
+  CellMeasurer,
+  CellMeasurerCache,
+  AutoSizer,
+} from 'react-virtualized';
 
-import space from 'app/styles/space';
-
+import {aroundContentStyle} from './styles';
 import ListHeader from './listHeader';
 import ListBody from './listBody';
-import {aroundContentStyle} from './styles';
+import {BreadcrumbsWithDetails} from './types';
+
+const LIST_MAX_HEIGHT = 400;
 
 type Props = {
   onSwitchTimeFormat: () => void;
-} & Omit<React.ComponentProps<typeof ListBody>, 'relativeTime'>;
-
-const List = React.forwardRef(
-  (
-    {
-      displayRelativeTime,
-      onSwitchTimeFormat,
-      orgId,
-      event,
-      breadcrumbs,
-      searchTerm,
-    }: Props,
-    ref: React.Ref<HTMLDivElement>
-  ) => (
-    <Grid ref={ref}>
-      <ListHeader
-        onSwitchTimeFormat={onSwitchTimeFormat}
-        displayRelativeTime={!!displayRelativeTime}
-      />
+  breadcrumbs: BreadcrumbsWithDetails;
+  relativeTime: string;
+} & Omit<React.ComponentProps<typeof ListBody>, 'breadcrumb' | 'isLastItem' | 'column'>;
+
+const cache = new CellMeasurerCache({
+  fixedWidth: true,
+  minHeight: 42,
+});
+
+class ListContainer extends React.Component<Props> {
+  componentWillReceiveProps() {
+    this.updateGrid();
+  }
+
+  componentDidUpdate() {
+    this.updateGrid();
+  }
+
+  wrapperRef = React.createRef<HTMLDivElement>();
+  listRef: List | null = null;
```

Comment:
Can we not use `createRef` here?

Reply:
unfortunately not, because if I pass a RefObject I don't get what I need :/